docs/docs/index.mdx

```diff
@@ -7,10 +7,10 @@ import { currentParams } from '@site/docs/governance/current-parameters.js';
 import { PlainVar } from '@site/src/js/Var';
 
 :::tip
-### **v14 Upgrade**
-Cosmos Hub will be upgraded to [v14.1](https://github.com/cosmos/gaia/releases/tag/v14.1.0) at block height: **[18,262,000](https://www.mintscan.io/cosmos/blocks/18262000)**
+### **v15.1 Upgrade**
+Cosmos Hub will be upgraded to [v15.1](https://github.com/cosmos/gaia/releases/tag/v15.1.0) at block height: **[19,639,600](https://www.mintscan.io/cosmos/blocks/19639600)**
 
-To upgrade from v13 check the [**upgrade guide**](/migration/cosmoshub-4-v14-upgrade.md)
+To upgrade from v14 check the [**upgrade guide**](/migration/cosmoshub-4-v15-upgrade.md)
```

Comment:
Link to https://github.com/cosmos/gaia/blob/v15.1.0/UPGRADING.md instead

Reply:
You have to copy the file into the docs folder. The root dir of gaia repo is not available in the docs

Reply:
But this link is available, right? https://github.com/cosmos/gaia/blob/release/v15.1.x/UPGRADING.md

Reply:
Let's do that for future releases after the docs refactor